classes/journal/JournalDAO.inc.php

```diff
@@ -116,7 +117,8 @@ public function anyPubIdExists(
         $forSameType = false
     ) {
         $pubObjectDaos = [
-            ASSOC_TYPE_ISSUE => DAORegistry::getDAO('IssueDAO'),
+            // TODO: See if this needs anything else or just updating like publication
+            ASSOC_TYPE_ISSUE => Repo::issue()->dao,
```

Comment:
Does anything else need to be considered here when listing the new DAO or is this alright as-is?

Reply:
I think it's alright as-is.